src/main/java/org/ibp/api/java/impl/middleware/inventory/InventoryServiceImpl.java

```diff
@@ -125,15 +126,19 @@ private static Integer getCurrentDateInt() {
 		return Integer.valueOf(new SimpleDateFormat("yyyyMMdd").format(Calendar.getInstance().getTime()));
 	}
 
+	private static Date getCurrentDate() {
```

Comment:
@vmaletta  in my opinion, there is no need to have a function for getting current date

Reply:
Ok